React/wbooks/src/app/screens/Login/components/LoginForm/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { Redirect } from 'react-router';
+
+import userIcon from '../../../../assets/user_icon.png';
+import passwordIcon from '../../../../assets/password.png';
+import { emailRegex, passwordRegex } from '../../../../../utils/regexs';
+import retrieveUserData from '../../../../../services/authService.js';
+
+import LoginError from './components/LoginError/index.js';
+import styles from './styles.scss';
+
+class LoginForm extends Component {
+  state = { name: '', password: '' , hasErrors: ''};
+
+  handleNameInput = event => {
+    this.setState({ name : event.target.value });
+  }
+
+  handlePasswordInput = event => {
+    this.setState({ password : event.target.value });
+  }
+
+  validateUser = async () => {
+    await retrieveUserData(this.state.name, this.state.password).then(() => {window.location.reload()})
```

Comment:
Reload?? That is wrong. Save in state component if is logged. then, in render method, render dynamically `Redirect` component or the login form. this approach will change when you use redux